sklearn/decomposition/_pca.py

```diff
@@ -28,7 +28,8 @@
 
 
 def _assess_dimension_(spectrum, rank, n_samples, n_features):
-    """Compute the likelihood of a rank ``rank`` dataset
+    """
+    Compute the likelihood of a rank ``rank`` dataset.
```

Comment:
It turns out that there was a issue in the numpydoc validator, and the previous version actually correct (cf https://github.com/numpy/numpydoc/issues/241 and https://github.com/scikit-learn/scikit-learn/pull/15451). If you use the script from current scikit-learn master it will ignore those GL01 checks. 

Could you please revert all such added newlines at the beginning of the docstring? Sorry about that.

Reply:
You got it.